sdks/python/apache_beam/io/fileio.py

```diff
@@ -702,9 +702,10 @@ def process(self, element, w=beam.DoFn.WindowParam):
 
     move_from = [f.file_name for f in temp_file_results]
     move_to = [f.file_name for f in final_file_results]
+
     _LOGGER.info(
-        'Moving temporary files %s to dir: %s as %s',
-        map(os.path.basename, move_from),
```

Comment:
originally this prints `Moving temporary files <map object at 0x7f8b1194d600> to dir` which was not helpful